maintenance/wikia/addLayoutToPortableInfoboxesAllWikias.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Script goes through all portable infoboxes definitions on all wikias and adds stacked layout if one is not defined.
+ * Stacked layout was default so far and with next release tabular layout will be default
+ *
+ * @author Kamil Koterba
+ * @ingroup Maintenance
+ */
+
+require_once( dirname( __FILE__ ) . '/../Maintenance.php' );
+
+/**
+ * Maintenance script class
+ */
+class addLayoutToPortableInfoboxesAllWikias extends Maintenance {
```

Comment:
We may not need this script at all if we choose the run-in-wiki-context model for the above script